src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.spec.data.ts

```diff
@@ -4380,6 +4380,48 @@ export let mockChartData = {
             'Number of QR codes with atleast 1 linked content': '6',
             'Number of QR codes with no linked content': '18'
         }
-    ]
+    ],
+    bigNumber: [{
```

Comment:
Even though you've added mock data, but there is no test case where you've used it. Please add some relevant test cases for the methods added.

Reply:
resolved